charts/grafana/values.yaml

```diff
@@ -195,9 +195,17 @@ gossipPortName: gossip
 ## or access it from within the cluster (ClusterIP service). Set the service type and the port to serve it.
 ## ref: http://kubernetes.io/docs/user-guide/services/
 ##
+
 service:
   enabled: true
   type: ClusterIP
+  loadBalancer:
+    enabled: true
+    type: LoadBalancer
+    status:
+      loadBalancer:
+        ingress:
+          - ip: 192.0.2.127
```

Comment:
@Sheikh-Abubaker 
I can't find these values being used:

https://github.com/search?q=repo%3Agrafana%2Fhelm-charts+loadBalancer.enabled+path%3A%2F%5Echarts%5C%2Fgrafana%5C%2F%2F&type=code

Reply:
@zanhsieh can you please explain what exactly is needed to be done because when I went through this documentation, I proceeded with just as it was mentioned there, correct me if I did something wrong, looking forward to learn and contribute to this issue.

![image](https://github.com/grafana/helm-charts/assets/99421476/7d378bd6-d2cb-4f89-ac63-4e8962fdeb77)

documentation link: https://kubernetes.io/docs/concepts/services-networking/service/#loadbalancer

Reply:
@Sheikh-Abubaker 
I think you have to
1.  Dig deeper how Helm chart rendering work
2. `status` stanza only appear during the runtime, not get managed by the helm chart render time.

In short, you just have to change the `type` under `service` from `ClusterIP` to `LoadBalancer` then it will arrange the IP once installed. The IP shall be distributed by the external load balancer; we don't have control unless your load balancer implementation is bare metal, e.g. MetalLB.

Reply:
@zanhsieh please take a look

![image](https://github.com/grafana/helm-charts/assets/99421476/eaaf4960-70c4-4c35-ad98-6fc10a4cec95)


I did changed type from ```ClusterIP``` to load ```LoadBalancer``` and also included parameter for ```loadBalancerSourceRanges```
  
but my doubt is that as we have changed type from ```ClusterIP``` to load ```LoadBalancer``` if there is a service now of type ```ClusterIP``` wouldn't we face errors now as there is now no parameters in ``` values.yaml``` for ClusterIP, or is it going to changes the type on it's own if ``` ClusterIP``` type is encountered as there is already the condition is defined below

![image](https://github.com/grafana/helm-charts/assets/99421476/a694d1ed-3ed9-4ba5-9a0b-b68c886fc856)